easybuild/easyconfigs/n/NGSadmix/NGSadmix-32-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CmdCp'
+
+name = 'NGSadmix'
+version = '32'
+
+homepage = 'http://www.popgen.dk/software/index.php/NgsAdmix'
+description = """This page contains information about the program called 
+NGSadmix, which is a very nice tool for finding admixture proportions 
+from NGS data. It is based on genotype likelihoods."""
```

Comment:
@sassy-crick Could you polish the description a little? ;)

Reply:
@vanzod I took the description more or less from their home page. I done some cosmetics to it though and I hope it reads better now.

Reply:
Let's also get rid of `This page contains information about the program called NGSadmix,`. Just start with `NGSadmix is a tool for finding[...]`